wheels/build/wheels.yml

```diff
@@ -49,8 +49,7 @@ packages:
     pysam:
         version: 0.12
         prebuild:
-            #wheel: sed -i '/\[bdist_wheel\]/a extra_compile_args = -D_GNU_SOURCE' ${SRC_ROOT}/setup.cfg && cat ${SRC_ROOT}/setup.cfg && for py in /python/cp*-`uname -m`; do ${py}/bin/easy_install 'cython' || exit 1; done
-            wheel: for py in /python/cp*-`uname -m`; do ${py}/bin/easy_install 'cython' || exit 1; done
+            wheel: sed -i -e 's/HTSLIB_MODE = .*/HTSLIB_MODE = "separate"/' ${SRC_ROOT}/setup.py; for py in /python/cp*-`uname -m`; do ${py}/bin/easy_install 'cython' || exit 1; done
```

Comment:
I think on OSX you need to change the `-i` option of `sed` to `-i.bak`

Reply:
Hrmm.  I can try that, but the failure looks like it is on the linux side.

Instead of -i.bak I actually usually do something like `sed -i '' 's/HTSLIB_MODE = .*/HTSLIB_MODE = "separate"/g' setup.py` on OSX and it works well without creating extra file copies.

Reply:
Unfortunately that syntax doesn't work on Linux, the only portable one creates the extra copy. Just forget it since it's not relevant here.

Reply:
@nsoranzo Got it, that makes sense, thanks for the tip!  I'll do that if it turns out the sed is broken.  Such a shame there's not a standard interface for it :)

(I just copied the `-i -e` syntax from other wheels that seem to be working okay, for reference)